### DISCUSSION TITLE
File Finder Does Not Match Open Tabs Outside Project

### DISCUSSION THREAD

**Discussion by pckilgore** [zed-industries/zed#24622] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

1. Open a project with a file.
2. Open other files outside the project (keymap and settings work good for this).
3. open the file finder (cmd + p)
4. see the open files from outside the project names in the list to start
5. search for one of the open files that is outside the project but still open in a tab ("set" for settings, if followed above).
6. Notice that the matching opened tab does not show up as a result.

### Environment

Zed: v0.143.7 (Zed)
OS: macOS 14.5.0
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by notpeter** [comment]
Normal files that are opened outside a project and untitled buffers are shown in search results.

Settings/keymap do not currently.
